app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -68,47 +75,29 @@ export default function ManageChannelMembers({
     currentTeamId,
     currentUserId,
     tutorialWatched,
+    teammateDisplayNameSetting,
 }: Props) {
     const serverUrl = useServerUrl();
     const theme = useTheme();
-    const {formatMessage} = useIntl();
+    const {formatMessage, locale} = useIntl();
 
     const searchTimeoutId = useRef<NodeJS.Timeout | null>(null);
     const mounted = useRef(false);
 
     const [isManageMode, setIsManageMode] = useState(false);
     const [profiles, setProfiles] = useState<UserProfile[]>(EMPTY);
+    const hasMoreProfiles = useRef(false);
     const [channelMembers, setChannelMembers] = useState<ChannelMembership[]>(EMPTY_MEMBERS);
     const [searchResults, setSearchResults] = useState<UserProfile[]>(EMPTY);
-    const [loading, setLoading] = useState(false);
+    const [loading, setLoading] = useState(true);
     const [term, setTerm] = useState('');
-
-    const loadedProfiles = (users: UserProfile[], members: ChannelMembership[]) => {
-        if (mounted.current) {
-            setLoading(false);
-            setProfiles(users);
-            setChannelMembers(members);
-        }
-    };
+    const [searchedTerm, setSearchedTerm] = useState('');
```

Comment:
I feel it is better if we do not change the result list until we actually "start searching". Adds less noise and less rendering.